contracts/interfaces/IIporProtocolAbi.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.16;
+
+import "../interfaces/types/AmmTypes.sol";
+import "./IAmmSwapsLens.sol";
+import "./IAmmPoolsLens.sol";
+import "./IAssetManagementLens.sol";
+import "./ILiquidityMiningLens.sol";
+import "./IPowerTokenLens.sol";
+import "./IAmmOpenSwapService.sol";
+import "./IAmmCloseSwapService.sol";
+import "./IAmmPoolsService.sol";
+import "./IAmmGovernanceService.sol";
+import "./IPowerTokenStakeService.sol";
+import "./IPowerTokenFlowsService.sol";
+
+/// @title Interface of the service that allows to close swaps.
```

Comment:
change description :slightly_smiling_face:

Reply:
and i think we don't need abi in name

Reply:
I found it before your comment :P fixed